src/components/certify/Certify.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div id="AR-step-4-container">
+  <div id="AR-step-4-container" :class="{'invalid-section': invalidSection}">
```

Comment:
The 'invalid-section' is missing from the original code.

Reply:
`invalid-section` probably should not be in the local code. It is an error class and, as such, should be declared globally (in assets folder) in every UI where this component is used (Storybook, Filings UI, etc). Probably it's already declared in most UIs.

Reply:
Agree with Severin, typically this is applied to a wrapper of sorts wherever locally implemented or if no section wrapper then on the component itself. 
ie `<article :class="{'invalid-section': invalidSection}"><component /></section>`
ie `<Certify :class="{'invalid-section': invalidSection}" />`

Reply:
Below are two places how the Create UI uses this component: [here](https://github.com/bcgov/business-create-ui/blob/c0ab60baa549c77d87d5a29482c8d6a7f6b120b7/src/views/Registration/RegistrationReviewConfirm.vue#L83-L94) and [here](https://github.com/bcgov/business-create-ui/blob/main/src/components/common/Certify.vue). I understand we can do `<Certify :class="{'invalid-section': invalidSection}" />`, but this style error bug still exists. 

![image](https://user-images.githubusercontent.com/51458187/185979353-8cf95a2d-7294-43c9-b830-50a22e5d5ee7.png)

![image](https://user-images.githubusercontent.com/51458187/185976804-f8b5567a-a5ac-4c43-8e03-fb2913d7d1dc.png)

Because the source code for this component is in BCRS, I created this ticket to fix this bug in this project.